modules/internal/UtilMisc_forDocs.chpl

```diff
@@ -44,31 +44,64 @@ Additional utilities
 */
 
 module UtilMisc_forDocs {
-  proc compilerError(param x:c_string ...?n, param errorDepth:int) {
+
+  /*
+     Generate a compile-time error.
+     The error text is a concatenation of the arguments.
+  */
+  proc compilerError(param x:c_string ...?n) {
     __primitive("error", (...x));
   }
 
   /*
-     These two functions are used primarily within system modules to signal
-     a compile time error.  The first version accepts a param int as the
-     last argument; this value is used to control the depth of an error
-     stack trace
+     Generate a compile-time error.
+     The error text is a concatenation of the string arguments.
+
+     :arg errorDepth: controls the depth of the error stack trace
   */
-  proc compilerError(param x:c_string ...?n) {
+  proc compilerError(param x:c_string ...?n, param errorDepth:int) {
     __primitive("error", (...x));
   }
 
-  proc compilerWarning(param x:c_string ...?n, param errorDepth:int) {
+
+  /*
+     Generate a compile-time warning.
+     The warning text is a concatenation of the arguments.
+  */
+  proc compilerWarning(param x:c_string ...?n) {
     __primitive("warning", (...x));
   }
 
-  /* Similar to compilerError() but generate a warning rather than an error */
-  proc compilerWarning(param x:c_string ...?n) {
+  /*
```

Comment:
I switched the two versions so the one without errorDepth now goes first, one with errorDepth - second. I believe I got those right, in line with the comments.